gnd/src/main/java/com/google/android/gnd/ui/common/BindingAdapters.java

```diff
@@ -50,6 +59,35 @@ public static void bindText(TextInputEditText view, Response response) {
     }
   }
 
+  @BindingAdapter("android:text")
+  public static void bindText(@NonNull TextView view, Date date) {
+    if (date == null) {
+      return;
+    }
+
+    DateFormat dateFormat = getMediumDateFormat(view.getContext());
```

Comment:
In retrospect, this logic really belongs in the ViewModel. Would it be possible to expose as an ObservableField<String> there? BindAdapters should be used sparingly or they'll quickly grown unmanageable otherwise. They're also relatively low level, implemented at the View rather than ViewModel layer.

Very sorry for flip-flopping on this one!

Reply:
This makes a lot more sense now. Thanks for the review! :smiley: